messages/install.txt

```diff
@@ -33,12 +33,12 @@ If you are on windows, or don't usually use git from the command
 line, there might be some things you'll need to set up. Go here
 for instructions on how to get set up:
 
-    https://docs.sublimegit.net/quickstart.html
+    https://github.com/SublimeGit/SublimeGit/blob/master/docs/quickstart.rst
```

Comment:
The documentation is available at readthedocs, which would be more appropriate: https://sublimegit.readthedocs.io/en/latest/quickstart.html